src/main/java/com/imaee/propinq/buildings/controllers/responses/BuildingResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.imaee.propinq.buildings.controllers.responses;
+
+import java.util.List;
+import java.util.UUID;
+
+public record BuildingResponse(
+        UUID buildingId,
+        String name,
+        String description,
+        String address,
+        List<String> imagesURL,
+        UUID userId,
+        UUID buildingTypeId
+) {
+}
```

Comment:
Acá una cuestión. Banco devolver esta información cuando estemos mostrando quizás el **detalle** del building, pero para la pantalla con el mapa y los marcadores hay información que no es necesaria y que incluso puede ralentizar las consultas a la base de de datos por el uso de JOINS que no van a terminar devolviendo nada útil.

Propongo que creemos dos DTOs: uno para devolver la lista de edificios que van a mostrarse en el mapa y otro para devolver el detalle. `BuildingResponse` y `BuildingDetailResponse`. Esto quedaría, entonces:

```suggestion
public record BuildingDetailResponse (
        String name,
        String description,
        String address,
        List<String> imagesURL,
        UUID userId, // Este ID es para acceder con un hipertexto al usuario (asumo que es el dueño/inmobiliaria)
        String userFullName, // En la pantalla vamos a mostrar el nombre del usuario
        String buildingTypeName // En la pantalla vamos a mostrar el nombre del buildingType
) {}
```

Por otro lado, para `BuildingResponse`

```
public record BuildingResponse (
        UUID buildingId,
        Locatable location, // Para devolver latitud y longitud
) {}
```

Reply:
Estos cambios no corresponden aplicarlos para esta Pull Request, deben aplicarse a la branch PROPINQ-55-PROPINQ-74